src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Compatibility.Common.targets

```diff
@@ -3,6 +3,7 @@
   <UsingTask TaskName="Microsoft.DotNet.PackageValidation.ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)" />
 
   <Target Name="RunPackageValidation"
+          DependsOnTargets="_GetPackageValidationReferenceDirectories"
```

Comment:
can we use property here like ```PackageValidationDependsOn``` ? We will need to provide this hook for our customers in order for them to do stuff with their packages like aspnetcore.

also do we need to provide an option for users to run without references ? I dont think of a scenario where we will need to run this without references. just bringing it up for discussion.